test/spec/dataviews/histogram-dataview-model.spec.js

```diff
@@ -18,6 +20,47 @@ describe('dataviews/histogram-dataview-model', function () {
     });
   });
 
+  it('should not listen any url change from the beginning', function () {
+    this.model.set('url', 'http://cartodb.com');
```

Comment:
What about this one @alonsogarciapablo?
If there is a url change, a fetch should happen (for rest of dataview models).